unity/Assets/Scripts/AgentManager.cs

```diff
@@ -508,7 +508,7 @@ private enum serverTypes { WSGI, FIFO };
 
         GameObject gameObject = GameObject.Instantiate(Resources.Load("ThirdPartyCameraTemplate")) as GameObject;
         gameObject.name = "ThirdPartyCamera" + thirdPartyCameras.Count;
-        gameObject.AddComponent(typeof(Camera));
+        //gameObject.AddComponent(typeof(Camera));
```

Comment:
Good point :)! Maybe just delete this line?

Reply:
will do!